tests/integration_tests/spark/sdk/clean_up_cluster.py

```diff
@@ -8,6 +8,7 @@ def clean_up_cluster(spark_client, id):
     try:
         cluster = spark_client.cluster.get(id)
         nodes = [node for node in cluster.nodes]
+        print([node.state for node in nodes])
```

Comment:
print?

Reply:
This is just in tests for debugging purposes. I can remove though.